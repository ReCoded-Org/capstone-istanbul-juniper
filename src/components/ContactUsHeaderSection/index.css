.contactUs__headerSection {
  background-color: #add4c5;
  margin: 0 auto;
  text-align: center;
  color: white;
}

.contactUs__headerSection__title {
  font-size: clamp(2.5rem, 3.5vw, 5rem);
  font-family: "Salsa", cursive;
  color: white;
  margin: 0 auto;
}

.contactUs__headerSection__context {
  font-family: "Raleway";
  font-size: clamp(1rem, 2vw, 2.5rem);
  color: white;
  margin: 0 auto;
}
