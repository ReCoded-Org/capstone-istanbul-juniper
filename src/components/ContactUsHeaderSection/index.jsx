import React from "react";
import "./index.css";
import { useTranslation } from "react-i18next";

const ContactUsHeaderSection = () => {
  const { t } = useTranslation();
  return (
    <div className="contactUs__headerSection">
      <h2 className="contactUs__headerSection__title">
        {t("contact.headerSection.title")}
      </h2>
      <p className="contactUs__headerSection__context">
        {t("contact.headerSection.subtitle")}
      </p>
    </div>
  );
};

export default ContactUsHeaderSection;
