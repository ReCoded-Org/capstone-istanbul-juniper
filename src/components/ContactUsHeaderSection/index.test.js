import React from "react";
import ContactUsHeaderSection from "./index.jsx";
import renderer from "react-test-renderer";

it("ContactUsHeaderSection snapshot test", () => {
  const tree = renderer.create(<ContactUsHeaderSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
