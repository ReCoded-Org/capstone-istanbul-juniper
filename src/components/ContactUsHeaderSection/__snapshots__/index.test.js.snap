// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUsHeaderSection snapshot test 1`] = `
<div
  className="contactUs__headerSection"
>
  <h2
    className="contactUs__headerSection__title"
  >
    contact.headerSection.title
  </h2>
  <p
    className="contactUs__headerSection__context"
  >
    contact.headerSection.subtitle
  </p>
</div>
`;
